### DISCUSSION TITLE
Check for: Code Indention

### DISCUSSION THREAD

**Discussion by aptMattKoe** [StefanMaron/BusinessCentral.LinterCop#28] [New Rule]
If the Code is not correct indented, it should be at least an warning.

**Comment by StefanMaron** [comment]
Is this really needed? Auto Format should be enabled and executed on save, then you never need to care about indent

**Comment by aptMattKoe** [comment]
Thats what I hope but I thought it would be great that the Linter could throw an warning for this :)
Not an must have in my eyes. Only something that comes up into my head :D

**Reply by christianbraeunlich** [reply]
One thing you might want to check out is [Prettier](https://github.com/prettier/prettier). To check all project files simply exectue the command `prettier --check .` or to automatically correct all Linter warnings `prettier --write .`

**Comment by StefanMaron** [comment]
I see. I will keep it in the backlog but wont look into it now, as I would expect this to not be that easy to write and the benefit is not that huge as by now.
